pkgs/by-name/sn/snowmachine/package.nix

```
@@ -1,16 +1,16 @@
-{ buildPythonPackage, lib, click, colorama, fetchPypi, setuptools-git }:
+{ buildPythonPackage, lib, click, colorama, hatchling, fetchPypi, setuptools-git }:
 
 buildPythonPackage rec {
   pname = "snowmachine";
-  version = "1.0.1";
+  version = "2.0.1";
+  pyproject = true;
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "1v385hhxy2a8vx5p0fhn0di8l4qfpb0a86j6nwsg0aw6ngb09qf1";
+    sha256 = "119e6da12f430af1519f1a9f091b77b7676c7a9dbeaab6616cb196fe793d8e61";
```

```suggestion
    hash = "119e6da12f430af1519f1a9f091b77b7676c7a9dbeaab6616cb196fe793d8e61";
```

you will also need to change to SRI hash notation

---

done